GameEngine/GameEngine/Engine/AssetManagement/Instance/MaterialInstance.hh

```diff
@@ -14,8 +14,6 @@ namespace AGE
 
 	struct MaterialInstance
 	{
-		//Key<Material> _material_key;
-		//std::map<std::string, Key<Property>> _properties;
```

Comment:
maybe can we change that be a typedef or using value instead of struct ? what do you think ?